src/hotspot/cpu/riscv/interp_masm_riscv.cpp

```diff
@@ -441,7 +441,13 @@ void InterpreterMacroAssembler::dispatch_base(TosState state,
                                               Register Rs) {
   // Pay attention to the argument Rs, which is acquiesce in t0.
   if (VerifyActivationFrameSize) {
-    Unimplemented();
+    Label L;
+    sub(t1, fp, esp);
```

Comment:
Note that RISC-V is a bit different in frame structure from other CPU platforms. I think we should exlcude the 2 frame metadata words after this `sub`, like:
```
sub(t1, t1, frame::metadata_words * wordSize); // Exclude 2 frame metadata words
```

Reply:
Thanks, I found that `sp -esp` equals 96, larger than `min_frame_size` 16. I think the extra 16 bytes is the `2 frame metadata words`.